docs/reference/airflow_tags.md

```diff
@@ -15,3 +16,11 @@ We borrow the [tiering system](https://wiki.mozilla.org/Sheriffing/Job_Visibilit
 - **impact/tier_1**: Highest priority/impact/critical DAG. A job with this tag implies that many downstream processes are impacted and affects Mozilla’s (many users across different teams and departments) ability to make decisions. A bug ticket must be created and the issue needs to be resolved as soon as possible.
 - **impact/tier_2**:  Job of increased importance and impact, however, not critical and only limited impact on other processes. One team or group of people is affected and the pipeline does not generate any business critical metrics. A bug ticket must be created and should be addressed within a few working days.
 - **impact/tier_3**: No impact on other processes and is not used to generate any metrics used by business users or to make any decisions. A bug ticket should be created and it’s up to the job owner to fix this issue in whatever time frame they deem to be reasonable.
+
+### triage/ tag
+
+This tag is meant to provide guidance to a triage engineer on how to respond to a specific DAG failure when the job owner does not want the standard process to be followed.
+
+- **triage/no_record**: Failures should _only_ be record and the job owner informed without taking any active steps to fix the failure.
```

Comment:
`triage/no_record` doesn't make sense to me.  Did you mean `triage/record_only`?

Reply:
you're right @sean-rose, it's a mistake and correcting it here: https://github.com/mozilla/bigquery-etl/pull/3589